docs/ldml/tr35-info.md

```diff
@@ -1239,6 +1239,13 @@ If any key-values are invalid, then they are ignored. Thus the following constru
 | -AA | invalid region 'AA'|
 
 ‡ Only the region portion is currently used, so in -rg-usabcdef the "abcdef" is ignored, whether or not it is valid.
+The table below shows when the region portion is valid or not.
+
+| Subdivision | Region | Used in -rg? | Comment |
+| --- | --- | --- | --- |
+| usca | US | Yes | Both the subdivision code (US Texas) and its region portion (US) are valid |
```

Comment:
```suggestion
| ustx | US | Yes | Both the subdivision code (US Texas) and its region portion (US) are valid |
```

unless I missed some news :)   - could also change Texas to California of course.

Reply:
fixing